tests/eko/evolution_operator/test_init.py

```diff
@@ -17,7 +17,7 @@
 
 
 def test_quad_ker_errors():
-    for p, t in [(False, True), (True, True)]:
+    for p, t in [(True, True)]:
```

Comment:
Replace also this one with suitable tests for implemented objects.

Reply:
Moreover, you can drop the `for` loop, and just inline the value of `p` and `t`.

This test is now specific for the polarized + time-like case.

Reply:
This is done